core/pom.xml

```diff
@@ -174,6 +174,11 @@
       <artifactId>lz4</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.roaringbitmap</groupId>
+      <artifactId>RoaringBitmap</artifactId>
+      <version>0.4.5</version>
```

Comment:
in fact the version should be removed here -- it should be inherited from the parent pom